readme.md

```diff
@@ -142,6 +145,11 @@ Resources that are not about specific concepts such as meetups, newsletters, etc
 - [Tykn newsletter](https://tykn.tech/subscribe-newsletter) - A monthly curated compilation of digital identity content.
 - [Verifiable Credentials 101 (SSI Meetup)](https://ssimeetup.org/verifiable-credentials-101-ssi-tyler-ruff-webinar-11/) - A SSI Meetup webinar by Tyler Ruff about VCs.
 - [SSI Wallets](https://github.com/Gimly-Blockchain/ssi-wallets) - An overview of SSI wallets, by [Gimly](https://gimly.io)
+- [Introduction to Digital Identity](https://walt.id/decentralized-identity-explained/digital-identity)
+- [Me, myself & (SS)I - co-authored by the Boston Consulting Group](https://static1.squarespace.com/static/609c0ddf94bcc0278a7cbdb4/t/623432727b5ccc4ad8876739/1647587956223/White+Paper+-+Self-Sovereign+Identity+%28SSI%29+%7C+BCG+x+walt.id.pdf)
+- [Introduction to NFTs for Identity](https://walt.id/decentralized-identity-explained/nfts)
+- [The Future of Decentralized Identity: SSI vs. NFTs](https://walt.id/decentralized-identity-explained/ssi-vs-nfts)
+- [The Pilot Playbook - How to build SS Pilots](https://walt.id/playbooks)
```

Comment:
Should this be SSI Pilots or is the SS on purpose?

Reply:
Well spotted, its a typo. Should be "SSI"